engine/dbc/data_enums.hh

```diff
@@ -1283,11 +1284,17 @@ enum spell_attribute : unsigned
   SX_DOT_HASTED                = 173u,
   SX_REQ_LINE_OF_SIGHT         = 186u,
   SX_DISABLE_PLAYER_MULT       = 221u,
+  SX_NO_DODGE                  = 247u,
+  SX_NO_PARRY                  = 248u,
+  SX_NO_MISS                   = 249u,
+  SX_NO_BLOCK                  = 256u,
   SX_TICK_MAY_CRIT             = 265u,
   SX_DURATION_HASTED           = 273u,
+  SX_DOT_HASTED_MELEE          = 278u,
   SX_FIXED_TRAVEL_TIME         = 292u,
   SX_SCALE_ILEVEL              = 354u,
   SX_ALLOW_CLASS_ABILITY_PROCS = 416u,
+  SX_REFRESH_EXTENDS_DURATION  = 436u,
```

Comment:
Why not encode the word Pandemic into this enum as well?

Reply:
Was just making it roughly match the flag text we standardized on from Blizzard, which in this case is "Periodic Refresh Extends Duration (436)"

Reply:
Ah ok. Well it seems there is no conversion to text output for spell attributes in spell query anyway, so it doesn't really matter. For anyone looking at it in the code base, the connection should be easy to establish.